packages/graphql-server/src/resolvers/identity-check/identity-check.d.ts

```diff
@@ -0,0 +1,61 @@
+export type Document = {
```

Comment:
These definitions are in TS Definitions right? Bit confused why we are copy-pasting them

Reply:
So we decide to maintain GraphQL server definition separately. In the future and we can map params on GraphQL server if services change any params name, we do not change to avoid breaking FE and it'll take more effort if we change schema and need to fix FE